interview-questions.md

```diff
@@ -15,54 +15,55 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
```

Comment:
These interview questions are an opportunity to speak on your experience (even if it is in class) with each topic. You are welcome to google an answer. You just have to paraphrase what you read. w3schools is one of my favorite places to research web development terms.